doc/conf.py

```diff
@@ -101,18 +100,13 @@
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-#
 # html_theme_options = {}
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
-html_context = {
-    'css_files': ['_static/theme_overrides.css',
-                  '_static/collapsible/collapsible_overrides.css']
-}
-
+# html_static_path = ['_static']
```

Comment:
Commented code?

Reply:
Well it is above? E.g. in line 103. I thought it all is a part of default config?